debian/rules

```diff
@@ -3,5 +3,5 @@
 export PYBUILD_NAME=aeneas
 
 %: 
-	dh $@ --with python2 --buildsystem=pybuild
+	dh $@ --with python2,python3 --buildsystem=pybuild
 
```

Comment:
Need one of these to resolve lintian error: `missing-dependency-on-numpy-abi`
```
override_dh_shlibdeps:
        dh_shlibdeps && dh_numpy && dh_numpy3
```
or
```
override_dh_python2:
	dh_python2
	dh_numpy

override_dh_python3:
	dh_python3
	dh_numpy3
```

Reply:
added.